client/signup/steps/user/index.jsx

```diff
@@ -163,4 +168,13 @@ export class UserStep extends Component {
 	}
 }
 
-export default localize( UserStep );
+export default connect(
+	( state ) => ( {
+		suggestedUsername: getSuggestedUsername( state )
+	} ),
+	( dispatch ) => ( {
```

Comment:
Dispatch will be mapped to action creators automatically, so:

```
( dispatch ) => ( {
	recordTrackEvent: ( event, data ) => {
		dispatch( recordTracksEvent( event, data ) );
	}
} )
```

can be shortened to just:

```
{ recordTracksEvent }
```